.github/workflows/style_type_checks.yml

```diff
@@ -9,22 +9,16 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: extractions/setup-just@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.8'
+    - uses: actions/setup-python@v4
     - name: Install dependencies
       run: |
         pip install .
-        pip install click black mypy
-        pip install types-python-dateutil
-        pip install types-waitress
-        pip install types-PyYAML
-    - name: Style and type checks
-      run: |
-        just black
-        just mypy
+        # todo: install also `black[jupyter]`
+        pip install click "black==24.01" "mypy==1.8.0" \
```

Comment:
I think it’s better to keep these to the latest version

Reply:
```suggestion
        pip install click "black==24.02" "mypy==1.8.0" \
```

Reply:
@Borda I meant to just use `black` without version specifier

Reply:
No, it is with specific pls see the actual PR's diff